### PR TITLE
Bringing Ember to the Desktop with NW.js

### PR DESCRIPTION
**Pull request by brzpegasus**

### CODE REVIEW

File: source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+```javascript
+console.log(window.process === global.process) // => true
+console.log(process.env.USER) // "brzpegasus"
+console.log(process.platform) // "darwin"
+```
+
+## Naming Conflicts
+
+Modules written with ES2015 syntax in your Ember app get transpiled into
```
Comment (bcardarella) [inline source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md]: maybe hyperlink to the es2015 project? I think some people still don't know es6 was renamed

File: source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+specify in your project's `package.json`:
+
+```json
+{
+  "name": "my-app",
+  "main": "dist/index.html"
+}
+```
+
+NW.js will parse this manifest on startup, launch a new Chromium browser
```
Comment (rwjblue) [inline source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md]: "parse this manifest", can you explain which manifest you mean? (you only mentioned `package.json` so far is that what you mean?)

Comment (brzpegasus) [inline source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md]: Yes, I'm referring to `package.json`. I can reword.

File: source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+window, then set the location to that starting page:
+`file:///Users/brzpegasus/projects/my-app/dist/index.html#/`.
+This does require that you set your `Ember.Router`
```
Comment (rwjblue) [inline source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md]: Maybe include the configuration for doing this in an ember-cli app?

File: source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+var fs = require('fs');       // Call core Node modules
+var async = require('async'); // Or even third-party modules!
+```
+
+If you're used to Node and CommonJS, this `require` function should look very
+familiar, but it isn't exactly the same. Here's what it does:
+
+```javascript
+function require(name) {
+  if (name == 'nw.gui')
```
Comment (rsocci) [inline source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md]: looks like you ate some brackets

Comment (brzpegasus) [inline source/posts/2015-03-26-bringing-ember-to-the-desktop-part1.md]: It's how it's actually implemented in NW.js. I wanted to preserve their code =).

**Pull request merged**